MAUI/DataGrid/Unbound-Column.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Unbound Columns in MAUI DataGrid control | Syncfusion
+description: Learn here all about Unbound columns support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Unbound column in MAUI DataGrid (SfDataGrid)
+
+An "Unbound column" typically refers to a column in a datagrid or table that is not directly bound to a data source. In other words, the data displayed in this column is not sourced from the underlying dataset but is rather calculated or manually entered. The unbound column can be added by using the SfDataGrid.DataGridUnboundColumn class.
+
+{% tabs %}
+{% highlight xaml %}
+
+     <syncfusion:SfDataGrid 
+                    x:Name="datagrid"    
+                    ColumnWidthMode="Fill"
+                    AutoGenerateColumnsMode="None"
+                    ItemsSource="{Binding OrderInfoCollection1}">
+     <syncfusion:SfDataGrid.Columns>
+         <syncfusion:DataGridUnboundColumn 
+                        MappingName="DiscountPrice"
+                        HeaderText="SUM"
+                        Expression="Price1+Price2"
+                        Format="C" />
+         </syncfusion:SfDataGrid.Columns>
+    </syncfusion:SfDataGrid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDataGrid dataGrid = new SfDataGrid();
+            OrderInforRepo orderInforRepo = new OrderInforRepo();
+            dataGrid.ItemsSource = orderInforRepo;
+            DataGridUnboundColumn DiscountColumn = new DataGridUnboundColumn()
+            {
+                MappingName = "DiscountPrice",
+                HeaderText = "SUM",
+                Expression = "Price1+Price2",
+                Format = "C"
+            };
+            this.datagrid.Columns.Add(DiscountColumn);
+            this.dataGrid.Columns.Add(DiscountColumn);
+
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows the outcome of the previous code:
+
+<img alt="" src="Images\unbound-column\maui-datagrid-sample-example.png" width="404" Height = 396/>
+
```

Comment:
Need to include the following note below like Xamarin UG,

<img width="665" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/60499432/a262bd11-2969-4ee2-822c-fb286fc06ef8">